src/main/jte/error/disabled_user.jte

```diff
@@ -14,7 +14,7 @@
         <div class="max-w-md">
             <h1 class="text-5xl font-bold">${context.localize("user.deactivated.title")}</h1>
             <p class="py-6">${context.localize("user.deactivated.message")}</p>
-            <a class="btn btn-primary" href="/">${context.localize("index.go")}</a>
+            <a class="btn btn-primary" href="/" aria-label="${context.localize("index.go")}" title="${context.localize("index.go")}">${context.localize("index.go")}</a>
```

Comment:
This text is already the text of the button, I don't think repeating it is necessary.